### DISCUSSION TITLE
Message: Error: Coordinator backup is corrupted

### DISCUSSION THREAD

**Discussion by azogtoth** [Koenkk/zigbee2mqtt#11767] [Devices]
Hi All,

All of a sudden, I get the above error message. I tried reflashing my Sonoff Zigbee 3.0 dongle, but no resolve. Does anybody have a suggestion or solution?

Thank you,
Stefan

> node index.js
Zigbee2MQTT:debug 2022-03-08 21:35:14: Loaded state from file /config/zigbee2mqtt/state.json
Zigbee2MQTT:info  2022-03-08 21:35:14: Logging to console and directory: '/config/zigbee2mqtt/log/2022-03-08.21-35-12' filename: log.txt
Zigbee2MQTT:debug 2022-03-08 21:35:14: Removing old log directory '/config/zigbee2mqtt/log/2022-03-08.21-26-12'
Zigbee2MQTT:info  2022-03-08 21:35:14: Starting Zigbee2MQTT version 1.24.0 (commit #unknown)
Zigbee2MQTT:info  2022-03-08 21:35:14: Starting zigbee-herdsman (0.14.20)
Zigbee2MQTT:debug 2022-03-08 21:35:14: Using zigbee-herdsman with settings: '{"adapter":{"concurrent":null,"delay":null,"disableLED":false},"backupPath":"/config/zigbee2mqtt/coordinator_backup.json","databaseBackupPath":"/config/zigbee2mqtt/database.db.backup","databasePath":"/config/zigbee2mqtt/database.db","network":{"channelList":[11],"extendedPanID":[221,221,221,221,221,221,221,221],"networkKey":"HIDDEN","panID":6754},"serialPort":{"path":"/dev/ttyUSB0"}}'
Zigbee2MQTT:error 2022-03-08 21:35:15: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2022-03-08 21:35:15: Failed to start zigbee
Zigbee2MQTT:error 2022-03-08 21:35:15: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2022-03-08 21:35:15: Exiting...
Zigbee2MQTT:error 2022-03-08 21:35:15: Error: Coordinator backup is corrupted
    at AdapterBackup.getStoredBackup (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/adapter-backup.ts:44:19)
    at ZnpAdapterManager.determineStrategy (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/manager.ts:134:24)
    at ZnpAdapterManager.start (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/manager.ts:63:26)
    at Controller.start (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:123:29)
    at Zigbee.start (/app/lib/zigbee.ts:58:27)
    at Controller.start (/app/lib/controller.ts:100:27)
    at start (/app/index.js:101:5)

**Comment by phobosdigg** [comment]
Last night after rebooting the raspberries got the same problem. Haven't changed any settings in weeks.

`[08:35:04] INFO: Handing over control to Zigbee2mqtt Core ...
> zigbee2mqtt@1.24.0 start
> node index.js
Zigbee2MQTT:error 2022-03-10 08:35:14: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2022-03-10 08:35:14: Failed to start zigbee
Zigbee2MQTT:error 2022-03-10 08:35:14: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2022-03-10 08:35:14: Exiting...
Zigbee2MQTT:error 2022-03-10 08:35:15: Error: Coordinator backup is corrupted
    at AdapterBackup.getStoredBackup (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/adapter-backup.ts:44:19)
    at ZnpAdapterManager.determineStrategy (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/manager.ts:134:24)
    at ZnpAdapterManager.start (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/manager.ts:63:26)
    at Controller.start (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:123:29)`

**Reply by azogtoth** [reply]
That is somehow comforting. I am not proficient enough to stand a chance solving this on my own. I've ordered a new Zigbee dongle, but it will take about a month getting that here... I hope somebody can point me in the right direction and get my Sonoff dongle working well before then.

**Reply by phobosdigg** [reply]
I wrote about the problem here [https://github.com/Koenkk/zigbee2mqtt/issues/11788](url)

**Comment by gonzzzales** [comment]
Have the same problem. Have you any solutions?

**Comment by azogtoth** [comment]
Sorry, no. I moved away from a zigbee USB dongle and modified my LIDL Zigbee controller, so now I can control my zigbee over Ethernet.

 

Good luck :(

 

Stefan

**Comment by davkri** [comment]
Hello !

For me, the problem was solved by deleting the zgbee2mqtt/data/coordinator_backup.json file.

After the grinding, the coordinator started for the first time.

My problem was caused either by a power outage or by connecting a flash drive.

Hi:
davkri

**Reply by deten** [reply]
This fixed it for me

**Reply by gonzzzales** [reply]
> Hello !
> 
> For me, the problem was solved by deleting the zgbee2mqtt/data/coordinator_backup.json file.
> 
> After the grinding, the coordinator started for the first time.
> 
> My problem was caused either by a power outage or by connecting a flash drive.
> 
> Hi: davkri

Thanks, it work!

**Reply by GottZ** [reply]
I confirm a power outage causes the coordinator backup to be entirely nullified.
<img width="644" height="111" alt="image" src="https://github.com/user-attachments/assets/3daeb1ae-73cb-4d35-b19c-a119c5ad6abf" />

**Reply by Koenkk** [reply]
@GottZ in case the file got corrupted, you can delete the coordinator_backup.json and z2m will re-create it on next startup

**Reply by GottZ** [reply]
> @GottZ in case the file got corrupted, you can delete the coordinator_backup.json and z2m will re-create it on next startup

thanks for the headsup. merely confirming the issue in late 2025.
maybe adding a coordinator_backup.json nullification check should be present on startup, to auto solve this?

**Reply by Koenkk** [reply]
I'm not sure if we want to blindly override it, I think in this case only the user can decide what should happen.

**Reply by GottZ** [reply]
> I'm not sure if we want to blindly override it, I think in this case only the user can decide what should happen.

tbh.. some people just install homeassistant and have no single clue where to find the log for zigbee2mqtt and simply think their bridge is broken.
This is ok for people that know how to google but certainly not for stable deployments.
At least improving the error message would be a reasonable thing to do.
